src/components/Button/Button.plugin.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const _ = require('lodash')
+const chroma = require('chroma-js')
+const defaultConfig = require('tailwindcss/defaultConfig')()
+
+function defaultOptions() {
+  return {
+    borderRadius: '1.5rem',
```

Comment:
use vales from the tailwind.js config here instead of declaring them explicitly this way we everything stays in sync and consistent

Reply:
Going to leave the default from tailwind default here and instead override these defaults using 'theme' vars in the plugin instantiation.